clojure-ts-mode.el

```diff
@@ -2032,6 +2032,22 @@ value is `clojure-ts-thread-all-but-last'."
                     "-"))))
     (user-error "No defun at point")))
 
+(defun clojure-ts-toggle-keyword-string ()
```

Comment:
That's inconsistent with `cycle` term used for privacy. (as we're again cycling between two states) I know where this is coming from, but I think we should do things better the second time around - let's use `clojure-ts-cycle-keyword-string` instead.